contrib/zpages/src/main/java/io/opencensus/contrib/zpages/ZPageHandlers.java

```diff
@@ -148,6 +147,7 @@ public static ZPageHandler getStatszZPageHandler() {
    * @since 0.6
    */
   public static void registerAllToHttpServer(HttpServer server) {
+    enableRunningSpanStore();
```

Comment:
Don't need to call it from here because getTracezPageHandler will be called and will enable this.

Reply:
Removed.